Makefile.in

```diff
@@ -49,8 +50,8 @@ INSTALL_DATA=install -m 644
 PWD=$(CURDIR)
 DESTDIR=
 bindir=$(prefix)/bin
-libdir=$(prefix)/lib
-incdir=$(prefix)/include
+libdir=$(prefix)/$(libdirsuffix)
```

Comment:
$libdir used to be something like /usr/local/lib/ this makes it be /usr/local/lib/ghdl by default. If this is good/bad is still up for debate.